ProcessMaker/processmaker
# Lock User Assignment

Resolves #2501

-  UI modified to store the assignmentLock


### To Test:
- Use the branch **develop** of the modeler project
-  create a process with a loop and validate that the user assigned for the first time to a task, when the flow returns to the same task, the assignment should be made to the same first user.
